x-pack/legacy/plugins/apm/public/components/app/ServiceMap/useRefHeight.ts

```diff
@@ -8,13 +8,17 @@ import { useWindowSize } from 'react-use';
 
 export function useRefHeight(): [
```

Comment:
This probably should be renamed to something like `useRefDimensions` since it gets height and width now.